### PR TITLE
DCS-2223 add segregation final check

### PR DESCRIPTION
**Pull request by louisenorris**
Check page and warning added to CA review:
Screenshot 2023-05-11 at 09 29 51

Screenshot 2023-05-11 at 09 29 59

DM review page also added:
Screenshot 2023-05-11 at 09 09 23

### CODE REVIEW

File: test/routes/viewModels/taskLists/tasks/finalChecks.test.js [hunk 1]
```diff
@@ -93,17 +100,24 @@ describe('final checks task', () => {
     test('should return multiple warning messages when multiple checks failed', () => {
```
Comment (GurnankCheema) [inline test/routes/viewModels/taskLists/tasks/finalChecks.test.js:100]: Do a test for segregation = no ?

**Review by GurnankCheema — APPROVED** [review]
LGTM
apart from comment re test for no segregation

File: test/routes/viewModels/taskLists/tasks/finalChecks.test.js (hunk 1 above)
Comment (louisenorris) [reply test/routes/viewModels/taskLists/tasks/finalChecks.test.js:102]: Adapted tests to check for this in latest commit 👍

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**